src/Output/Copy/Serializer/BodySerializer.php

```diff
@@ -14,7 +14,7 @@
 class BodySerializer
 {
     /** @var ContentTypeSerializerInterface[] */
-    private $contentTypeSerializers = [];
+    private array $contentTypeSerializers = [];
```

Comment:
if I'm not mistaken everything under /Copy/ will be copied to all clients, so we can't have php 7.4 specific feature here :(

Reply:
good catch, will change it